django_wikidata_api/fields.py

```diff
@@ -98,32 +144,55 @@ def to_wikidata_group(self):
         return ""
 
 
-
-class WikidataStringField(WikidataField):
-    pass
+class WikidataCharField(WikidataField):
+    """ WikidataItem Model field used to build and parse SPARQL queries as a string. """
+    serializer_field_class = serializers.CharField
 
 
-class WikidataLabelField(WikidataField):
-    suffix = 'Label'
-    serializer_field_class = serializers.CharField
+class WikidataLabelField(WikidataCharField):
+    """ WikidataItem Model field used to build and parse entity labels in SPARQL queries. """
     default_serializer_settings = {'allow_null': False, 'allow_blank': False}
+    service_property = "rdfs:label"
+    sparql_field_suffix = 'Label'
 
     def __init__(self, **kwargs):
         super(WikidataLabelField, self).__init__(**kwargs)
-        self.from_name = "{}{}".format(self.entity_name, self.suffix)
 
     def to_wikidata_field(self):
-        return "?{self.from_name} (?{self.from_name} AS ?{self.name})".format(self=self)
+        """
+        Get the portion of a SPARQL query that specifies the field name.
+        Returns (str):
+
+        """
+        return "{self.from_name} ({self.from_name} AS ?{self.name})".format(self=self)
 
     def to_wikidata_filter(self):
-        return ''  # Labels are not in the WHERE clause in a SPARQL query
+        """
+        Get the portion of a SPARQL query that specifies the filtering in the WHERE clause.
+
+        Notes:
+            Labels are not in the WHERE clause in a SPARQL query
+
+        Returns (""):
+
+        """
+        return ""
 
     def to_wikidata_service(self):
+        """
+        Get the portion of a SPARQL query that handles additional service information.
+        Returns (str):
+
+        """
         # TODO: Merge similarities with entity list label
-        return "?{self.entity_name} rdfs:label ?{self.from_name} . ".format(self=self)
+        return "?{self.entity_name} {self.service_property} {self.from_name} . ".format(self=self)
 
-    def to_wikidata_group(self):
-        return "?{self.from_name}".format(self=self)
+
+class WikidataDescriptionField(WikidataLabelField):
```

Comment:
This is the main contribution in this file.
Here I am creating a description field to add the `?itemDescription` argument to sparql queries like we do for  `?itemLabel`'s.